wolox-books/src/app/screens/auth/screens/book-detail/book-detail.component.scss

```diff
@@ -0,0 +1,46 @@
+@import '../../../../../sass/colors';
+@import '../../../../../sass/fonts';
+
+.container {
+  display: flex;
+  width: 100%;
+  max-width: 1100px;
+  background: $white;
+  padding: 20px;
+
+  .logo {
+    padding-right: 20px;
+    width: 400px;
+    height: 400px;
+    object-fit: contain;
+  }
+
+  .info {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    letter-spacing: 1px;
+
+    p {
```

Comment:
I think you arw not using a p tag but don't target html elements use classes instead

Reply:
P tag is not necessary now